components/CareerLink.tsx

```diff
@@ -0,0 +1,12 @@
+export default function CareerLink({ link }) {
```

Comment:
Sorry, I should have been more specific! This component can still live in the Companies.tsx file, but just should live outside of the Companies component